Core/Link.swift

```diff
@@ -37,17 +37,17 @@ public class Link: NSObject, NSCoding {
     public let url: URL
     public let localFileURL: URL?
     
-    public var displayTitle: String? {
+    public var displayTitle: String {
```

Comment:
I see 4 new warnings in the codebase now that this isn't optional - they were defaulting to the empty string, but don't need to do that any more.

* TabViewGridCell
* TabViewListCell